src/templates/career-details.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import Layout from '../components/layout/layout';
+import SEO from '../components/seo';
+import { Grid, GridItem } from '../components/grid/grid';
+import { PersonioJobPosition } from './career';
+import styled from 'styled-components';
+import { up } from '../components/breakpoint/breakpoint';
+import { TextTitle } from '../components/typography/typography';
+
+const Title = styled.h1`
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 110%;
+  color: #202840;
+
+  margin-top: 105px;
+  margin-bottom: 40px;
+
+  ${up('md')} {
+    margin-top: 145px;
+    font-size: 48px;
+  }
+`;
+
+const SectionTitle = styled(TextTitle)`
+  margin-bottom: 16px;
```

Comment:
`margin-top: 80px;` isntead of 160px for the career according to the design
![Screenshot 2021-03-25 at 08 46 45](https://user-images.githubusercontent.com/1701755/112436924-a4614f80-8d46-11eb-9862-49154fe80826.png)